src/azure-openapi-validator/tests/composite-azure-test.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { Message } from '../../jsonrpc/types';
+import { suite, test, slow, timeout, skip, only } from "mocha-typescript";
+import { safeLoad } from "js-yaml";
+import { AutoRestPluginHost } from "../../jsonrpc/plugin-host";
+import { run } from "../../azure-openapi-validator";
+import { AssertValidationRuleCount, CollectTestMessagesFromValidator, ReadFileAsString } from './utilities/tests-helper';
+
+
+@suite class CompositeAzureTest {
+  @test @timeout(120000) async "description should not be parameter name"() {
+    const file = 'src/azure-openapi-validator/tests/resources/DescriptionMustNotBeNodeName.json';
+    const openapiDefinitionDocument = ReadFileAsString(file);
+    const openapiDefinitionObject = safeLoad(openapiDefinitionDocument);
+    let messages: Message[] = await CollectTestMessagesFromValidator(file, openapiDefinitionObject);
+    AssertValidationRuleCount(messages, 'DescriptionMustNotBeNodeName', 1);
```

Comment:
Is there a way we could refer to the rule with something that allows a build type check? Being a string, won't tell us much, just wondering if we could have something similar to how to check on the type in C# side of things.